### PR TITLE
Diff for code review purposes.

### PR DESCRIPTION
**Pull request by wilatkabam**

### CODE REVIEW

**Comment by wilatkabam** [comment]
@jaygeeseman @FangAtKabam  Please let me know if you have any feedback! :+1:

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,31 @@
-markItUp! 1.1.14
+markItUp! 1.3
```
Comment (FangAtKabam) [inline CHANGELOG.md]: 1.3.1?

Comment (wilatkabam) [inline CHANGELOG.md]: http://sentimentalversioning.org/

File: markitup/jquery.markitup.js [hunk 2]
```diff
@@ -558,7 +558,7 @@
 				} else {
 					if (!template) {
 						$.ajax({
-							url: options.previewTemplatePath,
+							url: options.previewTemplatePath + document.getElementById('article_id').value,
```
Comment (jaygeeseman) [inline markitup/jquery.markitup.js]: If we're using jquery, `$('#article_id')` is more appropriate

File: markitup/sets/markdown/set.js [hunk 3]
```diff
@@ -0,0 +1,90 @@
+// -------------------------------------------------------------------
```
Comment (FangAtKabam) [inline markitup/sets/markdown/set.js]: do we consider apply standard code format for this file and css file at bottom?

**Comment by jaygeeseman** [comment]
Aside from the paths fix in bower.json, I'm not sure any of this should be in this repo. The markdown set was created for gatsby, so it should exist in gatsby. The rest of the customization can all be done inside the settings passed to `$(".markItUp").markItUp()`

**Comment by jaygeeseman** [comment]
Moving all these changes over to gatsby creates proper separation of responsibilities and removes circular references (I see at least two places these changes refer to gatsby). That way gatsby has full control over markitup without markitup having to know about what gatsby wants.

**Comment by wilatkabam** [comment]
@jaygeeseman are we talking about fully integrating markitup with Gatsby or only placing the additional settings there? I think markitup would have issues referring to assets like images that are not in its filepath.

**Comment by jaygeeseman** [comment]
I feel they should remain separate, including moving the markdown set into gatsby since it's a customization. Looking at the settings hash it seems to be possible to enter all the paths there and it should just work. Images are in the css file, which along with images should all be moved to gatsby. Paths should be no problem there.

**Comment by FangAtKabam** [comment]
yeah, since the setting is defined for gatsby, it is more appropriate to define settings in gatsby and pass the settings hash when you use markitup at [here](https://github.com/Kabam/gatsby/pull/137/files#diff-7169aa38d4c854e9055753e03e45e890R54) to decouple them and also css & images.

**Pull request closed without merging**